backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java

```diff
@@ -274,21 +278,25 @@ public Membership resolve(String qualifiedName) {
      */
     @Override
     public Membership resolveGlobal(String qualifiedName) {
+        Membership founded = null;
```

Comment:
I'm not sure but I think the name of the variable should be "found" withou "ed"

Reply:
Done